### ISSUE TITLE
Build for Android failing in both MacOs and Ubuntu

### ISSUE THREAD

**Issue by attermann** [BelledonneCommunications/linphone-sdk#81]
Building linphone-sdk is failing in both MacOS Catalina and Ubuntu 18.04 using latest master branch.  Seems that bctoolbox is not being included everywhere required.

cmake was configure with:

cmake -DLINPHONESDK_PLATFORM=Android -DCMAKE_BUILD_TYPE=ASAN ..

Below is the tail of build showing failure:

```
Scanning dependencies of target EP_bzrtp
[ 42%] Creating directories for 'EP_bzrtp'
[ 42%] No download step for 'EP_bzrtp'
[ 43%] No patch step for 'EP_bzrtp'
[ 43%] No update step for 'EP_bzrtp'
[ 43%] Performing configure step for 'EP_bzrtp'
loading initial cache file /home/chada/dev/linphone-sdk/build/WORK/android-arm64//tmp/EP_bzrtp/EP_bzrtp-cache-ASAN.cmake
-- Check for working C compiler: /usr/lib/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang
-- Check for working C compiler: /usr/lib/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/lib/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++
-- Check for working CXX compiler: /usr/lib/android-ndk/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for sqrt in m
-- Looking for sqrt in m - found
-- Found Sqlite3: /home/chada/dev/linphone-sdk/build/linphone-sdk/android-arm64/include  
-- Found XML2: /home/chada/dev/linphone-sdk/build/linphone-sdk/android-arm64/include/libxml2  
-- Configuring done
-- Generating done
-- Build files have been written to: /home/chada/dev/linphone-sdk/build/WORK/android-arm64/Build/bzrtp
[ 43%] Forcing build for 'EP_bzrtp'
[ 44%] Performing build step for 'EP_bzrtp'
Scanning dependencies of target bzrtp
[ 14%] Building C object src/CMakeFiles/bzrtp.dir/bzrtp.c.o
In file included from /home/chada/dev/linphone-sdk/bzrtp/src/bzrtp.c:22:
/home/chada/dev/linphone-sdk/bzrtp/include/bzrtp/bzrtp.h:23:10: fatal error: 'bctoolbox/crypto.h' file not found
#include "bctoolbox/crypto.h"
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.
src/CMakeFiles/bzrtp.dir/build.make:62: recipe for target 'src/CMakeFiles/bzrtp.dir/bzrtp.c.o' failed
make[8]: *** [src/CMakeFiles/bzrtp.dir/bzrtp.c.o] Error 1
CMakeFiles/Makefile2:103: recipe for target 'src/CMakeFiles/bzrtp.dir/all' failed
make[7]: *** [src/CMakeFiles/bzrtp.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make[6]: *** [all] Error 2
CMakeFiles/EP_bzrtp.dir/build.make:115: recipe for target '/home/chada/dev/linphone-sdk/build/WORK/android-arm64/Stamp/EP_bzrtp/EP_bzrtp-build' failed
make[5]: *** [/home/chada/dev/linphone-sdk/build/WORK/android-arm64/Stamp/EP_bzrtp/EP_bzrtp-build] Error 2
CMakeFiles/Makefile2:278: recipe for target 'CMakeFiles/EP_bzrtp.dir/all' failed
make[4]: *** [CMakeFiles/EP_bzrtp.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make[3]: *** [all] Error 2
CMakeFiles/android-arm64.dir/build.make:112: recipe for target 'android-arm64-prefix/src/android-arm64-stamp/android-arm64-build' failed
make[2]: *** [android-arm64-prefix/src/android-arm64-stamp/android-arm64-build] Error 2
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/android-arm64.dir/all' failed
make[1]: *** [CMakeFiles/android-arm64.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by Viish** [comment]
I think you didn't clone the linphone-sdk with --recursive option (or something went wrong with it).
Use `git submodule update --recursive --init` to fix that.

**Comment by attermann** [comment]
@Viish 

> I think you didn't clone the linphone-sdk with --recursive option (or something went wrong with it).

I don't think that' the case.  I actually tried four separate times with new repos downloaded from both github.com and gitlab.linphone.org, and on both MacOs and Ubuntu.  All exhibited the same behavior.

What I did find though is that when I leave off the `-DCMAKE_BUILD_TYPE=ASAN` param, then the problem disappears.  This is an acceptable workaround for me for the time being.
